### ISSUE TITLE
Side-by-side records in the same video

### ISSUE THREAD

**Issue by onderceylan** [justinribeiro/devtools-to-video#1]
Hey @justinribeiro, thanks for the great tool! 

Is there any plan to record 2 sites side-by-side in the same video? That would be a great feature imho.

**Comment by justinribeiro** [comment]
Yeah, I have a version of it that I haven't pushed yet (it needs a little refinement on the video placement and the stop frame), but I'll see about cleaning it up and pushing a version next week. :-)

**Comment by justinribeiro** [comment]
Just an update on this: I've been on the road quite a bit, but coming off later this week and then we'll see about getting this in. Apologizes for the delay!
